configure

```diff
@@ -22,9 +22,6 @@ function print_help {
   echo
   echo "  --install-python-toolchain        Install python in local virtualenv."
   echo  
-  echo "  --with-system-cmake (default)     Use system cmake, if avaliable."
```

Comment:
Probably want to bring this back in.

Reply:
Along with all the other cmake-related stuff and the shell script that builds it.

Reply:
Oops, I thought I had.  Good catch.